daphne_worker_test/tests/test_runner.rs

```diff
@@ -347,6 +353,44 @@ impl TestRunner {
         assert_matches!(task_config.vdaf, daphne::VdafConfig::Prio3(ref prio3_config) => {
             assert_matches!(prio3_config, daphne::Prio3Config::Sum{..})});
 
+        // Configure the endpoints
+        //
+        // First, delete the data from the previous test.
+        t.internal_delete_all(&t.batch_interval()).await;
+
+        // Configure the Leader with the Collector's bearer token.
+        t.leader_post_internal(
+            "/internal/test/add_authentication_token",
+            &json!({
+                "role": "collector",
+                "task_id": t.task_id.to_base64url(),
+                "token": t.collector_bearer_token.clone()
+            }),
+        )
+        .await;
+
+        // Configure the Leader with the Leader's bearer token.
+        t.leader_post_internal(
+            "/internal/test/add_authentication_token",
+            &json!({
+                "role": "leader",
+                "task_id": t.task_id.to_base64url(),
+                "token": t.leader_bearer_token.clone()
+            }),
+        )
+        .await;
+
+        // Configure the Helper with the Leader's bearer token.
+        t.helper_post_internal(
+            "/internal/test/add_authentication_token",
+            &json!({
+                "role": "leader",
```

Comment:
it shouldn't be helper? 
-- I understand the semantic here is setting the Leader's token that is used by the Helper. But as the code as it reads looks like something different.

Reply:
Hmm, yeah good point. I don't really have any bright ideas for making this clearer. We could maybe change "role" to "bearer_role"? However this would conflict with one of David's goals for the interop test runner: https://github.com/divergentdave/draft-dcook-ppm-dap-interop-test-design/pull/23/files#r996215019